brands/WBC/src/types.d.ts

```diff
@@ -4,6 +4,7 @@ declare module '@westpac/wbc' {
 		code: string;
 	};
 	type Colors = {
+		[key: string]: string;
```

Comment:
Colours needs to pre-defined set of key-pair values. So please remove this

Reply:
This line has been removed